src/components/MasterDetails.js

```diff
@@ -124,7 +129,7 @@ type MasterDetailsDefaultProps = {
 type MasterDetailsState = {
   /** The IDs of the selected rows */
   selectedRows: Array<string>;
-  /** The object for the most-recently selected row, to be displayed in the details pane.  */
+  /** The id for the most-recently selected row, to be displayed in the details pane. */
   detailsRowId: string;
```

Comment:
If the selectedRows array is always in chronological order, is the `detailsRowId` just here as a convenience? Would be be better to have a method to call to get the details row ID so we don't have to make sure we always keep these two fields in sync.

Reply:
https://github.com/attivio/suit/pull/88/files#r262257484